x/bep3/types/params.go

```diff
@@ -158,6 +169,20 @@ func validateBnbDeputyAddressParam(i interface{}) error {
 	return nil
 }
 
+func validateBnbDeputyFixedFeeParam(i interface{}) error {
+	bnbDeputyFixedFee, ok := i.(uint64)
+	if !ok {
+		return fmt.Errorf("invalid parameter type: %T", i)
+	}
+
+	// Sanity check (impossible for an unsigned integer to be negative)
+	if bnbDeputyFixedFee < 0 {
```

Comment:
nit: this is unnecessary, a better check might be
```suggestion
	if bnbDeputyFixedFee >= uint64(0) - uint64(1) {
```

or just omit it

Reply:
The proposed solution gives error `constant -1 overflows uint64`. Omitted the check entirely as the uint64 datatype already guarantees a non-negative number.